config/locales/instructions/en.yml

```diff
@@ -90,8 +90,9 @@ en:
         online accounts such as banks, email and social media.
       help_text_header: Password safety tips
       info:
-        lead: It must be at least %{min_length} characters long and not be a commonly
-          used password. That’s it!
+        lead: You need your 16-character personal key to reset your password if you
```

Comment:
It looks like this content is used on the sign-up page, so I think we need to keep both versions of the content.
https://github.com/18F/identity-idp/blob/a452fc25ffd724e0690342c00fab6d694da57d95/app/views/sign_up/passwords/new.html.erb#L5-L7

![image](https://user-images.githubusercontent.com/1430443/182896323-a63be169-9d06-4e92-b9d5-e157a5297484.png)

Reply:
ah, I had this on the wrong page 🤦🏾‍♀️

Reply:
There are a few places that people set passwords:

1. Reset Password (which doesn't include this content in any form currently)
2. Logged in and clicking "Edit Password"
3. Account Creation

|        1       |             2            |  3 |
|:--------------:|:------------------------:|:--:|
| ![image](https://user-images.githubusercontent.com/1430443/182903746-2ecfbb92-332e-4ead-ac28-875dff5e01f4.png) |  ![image](https://user-images.githubusercontent.com/1430443/182904034-2348208d-8a34-4be0-8825-37264273f889.png) | ![image](https://user-images.githubusercontent.com/1430443/182896323-a63be169-9d06-4e92-b9d5-e157a5297484.png) |

I think we want to:

1. Always show the new content
2. If the user is authenticated and verified, show the new content, otherwise show the current content
3. Always show the current content